sdk/typescript/test/e2e/utils/setup.ts

```diff
@@ -68,9 +70,15 @@ export function getClient(url = DEFAULT_FULLNODE_URL): SuiClient {
 export async function setup(options: { graphQLURL?: string; rpcURL?: string } = {}) {
 	const keypair = Ed25519Keypair.generate();
 	const address = keypair.getPublicKey().toSuiAddress();
+	setupClientConfig();
 	return setupWithFundedAddress(keypair, address, options);
 }
 
+export async function setupClientConfig() {
+	const configPath = path.join(homedir(), '.sui', 'sui_config', 'client.yaml');
+	execSync(`${SUI_BIN} client --yes --client.config ${configPath}`, { encoding: 'utf-8' });
+}
+
```

Comment:
SDK e2e tests need a `client.config` path to exist in the environment, where the `Move.lock` will be written to.

Reply:
can we use something other than the home directory for this?  Using a users config from their home directory seems like a bad idea of e2e tests (when running locally)

Reply:
eg, Either using a temp directory, or creating something in the repo with .gitignore to avoid checking it in

Reply:
Ah, I was basing this off of what we currently do in [here](https://sourcegraph.com/github.com/MystenLabs/sui@dd95851ea79ba88b8c9aba8d88aab9e86657ebec/-/blob/examples/trading/api/sui-utils.ts?L27-29) which uses `homedir()`, so it seemed reasonable to use that for client config too?

Reply:
I don't think that ever gets run as part of our e2e test suit, I think that is just an example app someone created, which is a little different

Reply:
Aha, makes sense, I'll look into it!